crates/trigger-http/src/lib.rs

```diff
@@ -254,7 +257,12 @@ impl HttpTrigger {
 
     /// Returns spin status information.
     fn app_info(&self) -> Result<Response<Body>> {
-        let info = AppInfo::new(self.engine.app());
+        let info = AppInfo {
+            name: self.engine.app_name.clone(),
+            version: self.engine.app().get_metadata(VERSION_KEY)?,
+            bindle_version: self.engine.app().get_metadata(BINDLE_VERSION_KEY)?,
+            oci_image_digest: self.engine.app().get_metadata(OCI_IMAGE_DIGEST_KEY)?,
```

Comment:
This change can be reverted. `oci_image_digest` is already populated by `AppInfo::new`